src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSession.kt

```diff
@@ -3,8 +3,8 @@ package de.bund.digitalservice.useid.identification
 import java.util.UUID
 
 data class IdentificationSession(
+    var eIDSessionId: UUID = UUID(0, 0),
```

Comment:
this assignment is meant for setting up initial value for the eIDSessionId, because `eIDSessionId` needs an initial value when we want to init the `IdentificationSession`, so that later in the future when we successfully fetched the `eIDSessionId` (from the tcToken endpont) we update its value. If we don't set an initial value, it throws an error in the `mockDataSource`. or is there a way to do that without assigning initial value?

Reply:
That's not good having magic values. Kotlin has something called nullable types.
```suggestion
    var eIDSessionId: UUID?
```

Reply:
yes, I had a look with the nullable type, but in the `MockDataSource` it throws an error
![image](https://user-images.githubusercontent.com/9930966/184391197-c80a1766-3715-4f9d-a313-387c74a3ee34.png)

which then I need to assign default value again in the `MockDataSource`, if not, the `MockDataSource` will throw an error same as above.

```kotlin
val identificationSession = IdentificationSession(
    eIDSessionId = UUID(0,0),
    refreshAddress = refreshAddress,
    requestAttributes = requestAttributes,
    useIDSessionId = useIDSessionId
)
```
Why UUID(0, 0) ?
https://stackoverflow.com/questions/26637225/how-can-i-create-an-empty-java-util-uuid-object

or maybe I am missing something? 🤔

Reply:
That should do the trick. The value should be null until it is used. Otherwise it's hard to tell if the ID is already set or not. 

```suggestion
    var eIDSessionId: UUID? = nul l,
```